.github/ci-failure-issue.md

```
@@ -0,0 +1,29 @@
+Un workflow CI a échoué sur `main`.
+
+## Run en échec
+
+- **Workflow** : {{ env.WORKFLOW }}
+- **Run** : [#{{ env.RUN_NUMBER }}]({{ env.RUN_URL }})
+- **Commit** : [`{{ env.SHORT_SHA }}`]({{ env.COMMIT_URL }}) par @{{ env.ACTOR }}
+- **Date** : {{ env.DATE }}
+
+## Jobs en échec
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)